### DISCUSSION TITLE
Is it possible to get latest build id and commit?

### DISCUSSION THREAD

**Discussion by RectangleEquals** [vercel/community#2352] [Help] [answered]
### Summary

I have a `vite` + `react-bootstrap` app running on Vercel, deployed automatically via GitHub, and I would find it _very_ useful to know within the app itself what the latest deployment build id and github commit id is, as displayed in the /deployments tab in the Vercel dashboard. Is such a thing possible?

**Comment by amyegan** [comment, marked as answer]
Hi @RectangleEquals! You can use the `VERCEL_GIT_COMMIT_SHA` [System Environment Variable](https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables#system-environment-variables) to get the commit id in your site code. And you can use the `VERCEL_URL` variable along with Vercel REST API to [get a deployment by deployment hostname](https://vercel.com/docs/rest-api/endpoints#get-a-deployment-by-id-or-url), which will return the id along with other deployment details.
